packages/client/base/src/services/hosted/v3/styles/crossmintHostedCheckoutV3StylesService.ts

```diff
@@ -0,0 +1,140 @@
+import type { Locale } from "@/types";
+import type {
+    CrossmintHostedCheckoutV3ButtonTheme,
+    CrossmintHostedCheckoutV3Props,
+} from "@/types/hosted/v3/CrossmintHostedCheckoutV3Props";
+import { t } from "@/utils/i18n";
+
+const BUTTON_CLASSNAME = "CrossmintHostedCheckoutButton";
+const BUTTON_LOGO_ID = "CrossmintHostedCheckoutLogo";
+
+const BUTTON_THEME_TO_CLASSNAME: Record<CrossmintHostedCheckoutV3ButtonTheme, string> = {
+    light: `${BUTTON_CLASSNAME}--Light`,
+    dark: `${BUTTON_CLASSNAME}--Dark`,
+    crossmint: `${BUTTON_CLASSNAME}--Crossmint`,
+};
+
+export function crossmintHostedCheckoutV3StylesService(hostedCheckoutProps: CrossmintHostedCheckoutV3Props) {
```

Comment:
similar to the appearance dom service in crossmint-main for embed v3

doing the styles like this will allow us to have 1 service across all the frameworks, (react, vue, vanilla, etc)